### PR TITLE
Added conditionals around sysadmin plugin

### PR DESCRIPTION
**Pull request by shaidar**
#### What's this PR do?
Adds some conditionals to deploy sysadmin plugin to Lilac release.

### CODE REVIEW

File: pillar/edx/ansible_vars/residential.sls [hunk 1]
```diff
@@ -130,8 +130,10 @@ edx:
       - name: eox-core[sentry]
       - name: git+https://github.com/raccoongang/xblock-pdf.git@8d63047c53bc8fdd84fa7b0ec577bb0a729c215f#egg=xblock-pdf
         extra_args: -e
-      # edX Sysadmin plugin
-      #- name: https://github.com/mitodl/edx-sysadmin#egg=edx-sysadmin
+      {% if 'koa' not in grains.get('edx_codename') %}
```
Comment (blarghmatey) [inline pillar/edx/ansible_vars/residential.sls:133]: We should just make this the default and not add the conditionals.

File: pillar/edx/ansible_vars/residential.sls [hunk 2]
```diff
@@ -130,8 +130,10 @@ edx:
...
+      edX Sysadmin plugin
```
Comment (blarghmatey) [inline pillar/edx/ansible_vars/residential.sls:134]: Is this intended to be a comment?

File: pillar/edx/ansible_vars/residential.sls [hunk 3]
```diff
@@ -174,7 +174,11 @@ edx:
         RESTRICT_ENROLL_SOCIAL_PROVIDERS:
           - mit-kerberos
         ENABLE_THIRD_PARTY_ONLY_AUTH: True
+        {% if 'koa' not in grains.get('edx_codename') %}
```
Comment (blarghmatey) [inline pillar/edx/ansible_vars/residential.sls:177]: We don't need a conditional here either.

Comment (shaidar) [reply pillar/edx/ansible_vars/residential.sls:177]: Why not?

Comment (blarghmatey) [reply pillar/edx/ansible_vars/residential.sls:177]: I guess it doesn't matter either way.

**Comment by blarghmatey** [comment]
:+1:

**Pull request merged**